modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/MultiStepNav.es.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayMultiStepNav from '@clayui/multi-step-nav';
+import React, {useState} from 'react';
+
+import Sheet from './Sheet.es';
+
+const MultiStepNav = ({steps}) => {
+	const [value] = useState(0);
```

Comment:
I'm imagining that is not implemented the index yet, right? that way we could move to a simple variable for now

When the times come to set a new value, this could be a state again, what you guys think ? @jairmedeiros , @silasbsi 

```suggestion
	const value = 0;
```

Reply:
This value could be also named **currentStep**, since **value** is a generic name

```suggestion
	const currentStep = 0;
```

Reply:
I agree!